.eslintrc.json

```diff
@@ -1,3 +1,23 @@
 {
-  "extends": "eslint-config-edx"
+  "extends": "eslint-config-edx",
+  "globals": {  // Try to avoid adding any new globals.
```

Comment:
Note to reviewers: Normally comments aren't allowed in JSON, but ESLint uses a non-standard JSON parser that specifically allows comments. That's why it's all red in GitHub.